lib/tasks/workflow_counters.rake

```diff
@@ -0,0 +1,11 @@
+desc 'Update counter caches for workflows'
+namespace :counters do
+  namespace :rebuild do
+    task workflows: :environment do
+      Workflow.reset_column_information
```

Comment:
What is the purpose of this call? The schema hasn't changed, what needs to be reloaded?

Reply:
That's normally only needed if you modify the schema and then try to use it in the same process. Perhaps you were running Spring and thus had an already-running-process with the old schema loaded? Not sure, cause Spring shouldn't have loaded the app up to the point where it's loaded the schema.